systemtests/custom-operator-registry/build_push_registry.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+FROM=${1}
+CUSTOM_IMAGE=${2}
+
+rm -rf manifests
+id=$(docker create $FROM)
```

Comment:
Can you allow override the docker command using an env , I.e ${DOCKER}?

Reply:
sure :)